scripts/setup/validate_pipeline.py

```
@@ -0,0 +1,50 @@
+        checks.append(("✅", "스키마 추출 완료"))
+    else:
+        checks.append(("❌", "스키마 추출 필요: uv run python scripts/setup/export_schema.py"))
+    
+    # 2. 비즈니스 컨텍스트 파일 확인
+    yaml_file = Path("scripts/data/business_context/tables.yaml")
+    if yaml_file.exists():
+        checks.append(("✅", "비즈니스 컨텍스트 파일 존재"))
+        
+        # 3. JSON에 비즈니스 정보 적용 확인
+        import json
+        sample_json = list(schema_dir.glob("*.json"))[0]
```

Ruff (0.13.1)

Prefer `next(iter(schema_dir.glob("*.json")))` over single element slice

Replace with `next(iter(schema_dir.glob("*.json")))`

(RUF015)